scripts/solution_view.py

```diff
@@ -33,14 +33,20 @@ class SimpleAdapter(TabularAdapter):
 
 class SolutionView(HasTraits):
   python_console_cmds = Dict()
-
+#we need to doubleup on Lists to store the psuedo absolutes separately without rewriting everything
```

Comment:
Indent comment to match code and put a space after the `#`. Also, wrap to 80 chars